rllib/tuned_examples/ppo/multi_agent_pendulum_ppo.py

```diff
@@ -16,7 +16,7 @@
         lr=0.0003,
         sgd_minibatch_size=64,
         num_sgd_iter=10,
-        model={"fcnet_hiddens": [128, 128], "fcnet_activation": "relu"},
```

Comment:
We previously made this change `"fcnet_activation": "relu"` because it improved performance, but it did not tackle the root cause.